loops.js

```diff
@@ -7,6 +8,14 @@
 // ****
 // *****
 
+for(i = 0 ; i < 6 ; ++ i) {
+    stars = ''
+    for(j = 0; j < i; ++j) {
+stars += '*'
+    }
+    console.log(stars);
```

Comment:
Een loop in een loop was hier niet persé nodig. Je had hier ook de repeat methode kunnen gebruiken!

console.log('*'.repeat(i + 1));